cmd/protoc-gen-cpp-tableau-loader/go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/tableauio/loader/cmd/protoc-gen-cpp-tableau-loader
+
+go 1.21
+
+replace github.com/tableauio/loader => ../..
```

Comment:
`go install` not support `replace`. See https://github.com/tableauio/tableau/pull/181

Reply:
I will close this PR, just release by the main module version.